packages/components/src/split-button/README.md

```diff
@@ -0,0 +1,46 @@
+SplitButton
```

Comment:
The split button example pops down under the WP menu

<img width="412" alt="Screen Shot 2019-09-06 at 11 36 45 AM" src="https://user-images.githubusercontent.com/343847/64436478-cf1f8080-d09a-11e9-8970-e3fa05735eac.png">

Reply:
Mind if we punt on this? It occurs on `master` as well. (Same for `ViewMoreList`)

Reply:
Sounds good.